Added heartbeat checking to ps_to_http

mynameisfiber opened the pull request:
ps_to_http now takes in a max_silence parameter which denotes the
maximum time (in seconds) between messages before quitting.  This is
intended to deal with source pubsubs that are hanging and not sending
messages (or subsequently need a reconnection).

jehiah commented on ps_to_http/ps_to_http.c:
```diff
@@ -73,6 +94,15 @@ void process_message_cb(char *message, void *cb_arg)
+    if (option_get_int("max_silence") > 0) {
+        if (last_message_timestamp == 0) {
+            _DEBUG("Registering timer.\n");
+            evtimer_set(&silence_ev, silence_cb, NULL);
```
i think this initialization of the timer should move to `main()`. As it is now, it will not fire if a connection never receives a message.

jehiah commented on ps_to_http/ps_to_http.c:
```diff
@@ -62,6 +65,24 @@ void finish_destination_cb(struct evhttp_request *req, void *cb_arg)
+    _DEBUG("Testing for infinite silence\n");
+    if ( time(NULL) - last_message_timestamp > max_silence_time.tv_sec ) {
+        _DEBUG("Things are too quiet... time to quit!\n");
+        error_cb(-127, (void*)NULL);
```
it would probably be good to give a little context here beyond `_DEBUG` since this is an abnormal exit. `fprintf(stderr, "exiting; no message received for %d seconds (limit %d seconds)\n"....)`

jehiah commented:
also can you bump the `#define VERSION` in ps_to_http.c and update the `README.md`

Commit:
Added heartbeat checking to ps_to_http

ps_to_http now takes in a max_silence parameter which denotes the
maximum time (in seconds) between messages before quitting.  This is
intended to deal with source pubsubs that are hanging and not sending
messages (or subsequently need a reconnection).

mynameisfiber commented:
@jehiah I took care of the comments and even updated the README.md (documentation?! how novel!)
